src/modules/role/role.service.ts

```diff
@@ -46,8 +48,26 @@ export class RoleService {
    * return true if role with given namespace exists
    * @param namespace
    */
-  public async exists(namespace: string) {
-    return Boolean(await this.getByNamespace(namespace));
+  public async exists(
```

Comment:
This signature does not allow to check only parent organization

Reply:
This exists method is for the `roles`  i don't think there's a reason for us to be checking for only a parentOrg here, we can either check only for a role namespace or role namespace belonging to a parentOrg. we can check for only parent org namespace in the exists method for organisation.

Reply:
I mean that optional arguments can be omitted, but `parentApp` can't be omitted, when you want to filter only by `parentOrg`